framework/scripts/wazuh_server.py

```diff
@@ -131,17 +133,21 @@ def start_daemons(background_mode: bool, root: bool):
     root : bool
         Whether the script is running as root or not.
     """
-    daemons = {
-        ENGINE_DAEMON_NAME: [ENGINE_BINARY_PATH, 'server', 'start'],
-        MANAGEMENT_API_DAEMON_NAME: [EMBEDDED_PYTHON_PATH, MANAGEMENT_API_SCRIPT_PATH]
+
+    daemons = [
+        {'name': ENGINE_DAEMON_NAME, 'args': [ENGINE_BINARY_PATH, 'server', 'start']},
```

Comment:
Didn't notice it in the previous review, but what's the advantage of using a list of dictionaries instead of a dictionary? I would just keep this unchanged

```suggestion
    daemons = [
        {'name': ENGINE_DAEMON_NAME, 'args': [ENGINE_BINARY_PATH, 'server', 'start']},
```

Reply:
With Python 3.7+, a dictionary is guaranteed to be iterated in the insertion order of keys ([ref](https://stackoverflow.com/questions/39980323/are-dictionaries-ordered-in-python-3-6)).

If we keep a list, we need to update the unit test.

Reply:
Taking advantage of the changes required in other comments, let's maintain the use of the dictionary.